singer-io/tap-mambu
# Release/32

# Description of change
Added sorting to journal lines endpoint config, which fixes an extraction issue where the API would miss some records (due to sorting not being specified)

# Manual QA steps
 - Manual regression test (to check that tap output is not impacted)
 - Manually tested that all records are extracted (including those skipped)
 - Tap Tester passed
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
